Makefile

```diff
@@ -10,12 +10,20 @@ cidocker = docker run ${ciargs} ${dockerargs} golang:1.4
 all: ${build_image}.created build check integration-tests
 
 integration-tests:
+	@if [ ! -e ./cmd/dnet/dnet ]; then \
+		echo "Dnet binary not present, triggering build"; \
+		make build; \
+	fi
 	@if [ ! -d ./integration-tmp ]; then \
 	    mkdir -p ./integration-tmp;	\
 	    git clone https://github.com/sstephenson/bats.git ./integration-tmp/bats; \
 	    ./integration-tmp/bats/install.sh ./integration-tmp; \
 	fi
 	@./integration-tmp/bin/bats ./test/integration/dnet
+	@if [ -e ./cmd/dnet/dnet ]; then \
```

Comment:
Ok, given go builds are faster I am fine with removing it once the integration-tests are done.